__tests__/hooks/useGroupSetting.test.js

```diff
@@ -18,6 +17,12 @@ jest.mock('react-native-simple-toast', () => ({
   show: jest.fn()
 }));
 
+jest.mock('react-native-compressor', () => {
+  return {
+    compress: jest.fn(() => 'file:///imag.jpg')
+  };
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new mock for 'react-native-compressor' has been added. This seems fine as long as the mocked function returns what's expected in the actual tests.
```diff
+ jest.mock('react-native-compressor', () => {
+   return {
+     compress: jest.fn(() => 'file:///imag.jpg')
+   };
+ });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->